libs/vkd3d/command.c

```diff
@@ -814,137 +476,12 @@ static void d3d12_fence_destroy_vk_objects(struct d3d12_fence *fence)
 
     vk_procs = &device->vk_procs;
 
-    for (i = 0; i < ARRAY_SIZE(fence->old_vk_fences); ++i)
-    {
-        if (fence->old_vk_fences[i])
-            VK_CALL(vkDestroyFence(device->vk_device, fence->old_vk_fences[i], NULL));
-        fence->old_vk_fences[i] = VK_NULL_HANDLE;
-    }
-
-    d3d12_fence_garbage_collect_vk_semaphores_locked(fence, true);
-
     if (fence->timeline_semaphore)
```

Comment:
This check can be removed as well, even if the object is `null`.

Reply:
Should be fixed now.